### PR TITLE
[#1] [Chore] Init the project

### LINKED ISSUE #1: [Chore] Init the project
## Why
To quickly kick-off a Flutter project, we use Nimble's [Flutter templates](https://github.com/nimblehq/flutter-templates).

## Acceptance Criteria
Generate a new project using the Flutter template with following parameters:
- Package name is `co.nimblehq.kaungthieu.survey`
- App name is `Survey Flutter`

### LINKED ISSUE #2: [Chore] Set up the project Code owners
## Why
We need to update CODEOWNER with the assigned developers from our team.

## Acceptance Criteria
- Update CODEOWNER with the following:
  - Member:  `@kaungkhantsoe`, and `@Thieurom`
  - Code Reviewer: to be assigned.
  - Engineering Lead: `@minhnimble`

### PR DESCRIPTION
**Pull request by Thieurom**
- Closes #1
- Closes #2

## What happened
Init the project from [flutter-templates](https://github.com/nimblehq/flutter-templates).

## Insight
- Use Flutter 3.7.8 with `fvm`.
- The project parameters:
  - PACKAGE_NAME=co.nimblehq.kaungthieu.survey
  - PROJECT_NAME=survey_flutter
  - APP_NAME="Survey Flutter"
- Remove unused Python virtual environment and scripts.
- Update README to this project instead of generated one.
- Update CODEOWNERS.

## Proof Of Work
|iOS|Android|
|---|---|
|Simulator Screen Shot - iPhone 14 - 2023-03-29 at 10 21 38|Screenshot_20230329_100919|

SCR-20230329-mynb

### CODE REVIEW

**Review by kaungkhantsoe — DISMISSED** [review]
I am gonna approve since I could run the app. It is great to have more clearer instructions in README file 👍

File: lib/l10n/app_vi.arb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (manh-t) [inline lib/l10n/app_vi.arb:1]: @Thieurom we can remove the unused translations in the scope of IC 😄

**Review by manh-t — APPROVED** [review]
Just one suggestion, hence approved

File: lib/l10n/app_vi.arb (hunk 1 above)
Comment (Thieurom) [reply lib/l10n/app_vi.arb:1]: @manh-t
I saw that we have 3 languages, `en`, `th`, and `vi`.
If we remove the `vi`, should we remove the `th` as well, then keep only the `en`?

Comment (manh-t) [reply lib/l10n/app_vi.arb:1]: @Thieurom it's okay to keep only `en` 🙆‍♂️

Comment (Thieurom) [reply lib/l10n/app_vi.arb]: Removed `vi` and `th` localized bundles in https://github.com/nimblehq/flutter-ic-kaung-thieu/pull/34/commits/9cd3345e00a09071dc56bcaefa3846584c2d552e
@kaungkhantsoe and @manh-t, please help me check and approve the PR again 🙏

**Review by manh-t — APPROVED** [review]
🚀

**Pull request merged**